org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/entitytypes/properties/PropertiesDefinitionResource.java

```diff
@@ -75,6 +78,59 @@ public PropertiesDefinitionResourceApiData getJson() {
         return new PropertiesDefinitionResourceApiData(this.getEntityType().getProperties(), this.wpd);
     }
 
+    // TODO: implement getMerged endpoint
+    // TODO: do we require deep copies?
+    // TODO: this merges only wineryPropertiesDefinitions?
+    // TODO: do we need to merge entityType#getProperties or something else?
+    // TODO: does not know at the end from which entityType the property definition has been derived from
+    @GET
+    @Path("merged")
+    @Produces(MediaType.APPLICATION_JSON)
+    public PropertiesDefinitionResourceApiData getMerged() {
+        ArrayList<TEntityType> parents = RepositoryFactory.getRepository().getParents(this.parentRes.getEntityType());
+        
+        // NOTE: this is not a deep copy but a reference!
```

Comment:
If you want to have a dedicated list, you can use `new ArrayList(this.getEntityType()....);`
However, since this is only used to return the elements, this is okay. In case you would use it as an utils method, this is a valid point.